yt_dlp/extractor/bitchute.py

```diff
@@ -126,7 +126,7 @@ def _real_extract(self, url):
             'title': self._html_extract_title(webpage) or self._og_search_title(webpage),
             'description': self._og_search_description(webpage, default=None),
             'thumbnail': self._og_search_thumbnail(webpage),
-            'uploader': clean_html(get_element_by_class('owner', webpage)),
+            'uploader': self._search_regex(r'<p\sclass=\"name.+Channel\">([^<]+)<', webpage, 'uploader'),
```

Comment:
How about something like this:
```suggestion
            'channel': clean_html(get_element_by_class('name', details)),
            'uploader': clean_html(get_element_by_class('creator', details)),
```
with
```python
details = get_element_by_class('details', webpage) or ''
```
Potentially, the `channel_url` and `uploader_url` could also be extracted, as these can also differ.

Reply:
I did a couple of changes, and added the `channel_url` and `uploader_url` as you suggested.

Reply:
I think both `channel` and `uploader` should be added as they can differ as seen [here](https://www.bitchute.com/video/Yti_j9A-UZ4/).

Reply:
Thanks for the example @seproDev . I added the `channel` extraction as well, since the information was already there.